Automattic/jetpack
# Story block: Make stories available in the block inserter

#### Changes proposed in this Pull Request:
This is the release PR for the story block. It just flips the switch and makes the block available from the inserter in production.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Run `yarn build-production`
* Load the editor, make sure the story block is available from the inserter and works as expected
